docs/user-guide/gaffer-basics/what-is-cardinality.md

```diff
@@ -5,8 +5,13 @@ Gaffer queries.
 
 ## What is Cardinality?
 
-In Gaffer, cardinality represents the number of unique vertices that are connected to a given
-vertex.
+Cardinality is a mathematical term which translates to the number of elements contained in a set.
+In graphical databases it can be used to refer to the relationships between data. 
+In Gaffer, by adding a cardinality entity to your graph, vertices will inherit an estimate of the number of 
+unique connections between given entities.
```

Comment:
When this page was originally added to the docs, I had some concerns around the phrasing of this section as the use of 'cardinality' in Gaffer doesn't line up perfectly with the usual definition. [See here for my original comments](https://github.com/gchq/gaffer-doc/pull/351#pullrequestreview-1474360685).

While this expands the level of detail, I think it needs to be clearer that how cardinality is used in Gaffer is not quite the same as the mathematical definition.

Reply:
Ah that is actually helpful and makes a lot of sense. I didn't quite know how detailed to go but I can add in more along the lines of your original comments.